skbio/io/__init__.py

```diff
@@ -246,6 +246,7 @@ class of Python's standard library. The goal of a ``sniffer`` is two-fold: to
     GFF3FormatError,
     EMBLFormatError,
     BIOMFormatError,
+    EmbedFormatError,
```

Comment:
It turned out that this wasn't even being imported -- hinting that we don't have tests that verify any of these errors. 
Raising an issue, since this could be a blind spot when creating new file formats.